src/containers/editor/graph/Toolbar.tsx

```diff
@@ -59,12 +61,10 @@ const Toolbar = memo(({ id }: ToolbarProps) => {
     setFoldSiblings(!foldSiblings);
   };
 
-  // TODO: hide fold button when there is no edges
-  // TODO: fix w-fit doesn't work
   return (
     <NodeToolbar
       isVisible={true}
-      className="flex items-center justify-center w-[96px] h-fit bg-input"
+      className="flex items-center justify-center w-fit h-fit bg-input gap-x-1"
```

Comment:
Why do you want to add space between buttons?

Reply:
I changed it to h-6 and w-6 to cover the padding, because fit-content doesn't consider the padding of each child and as a result, the container's the whole size is smaller than the actual size of children.